projects/prime-protocol/index.js

```diff
@@ -12,60 +12,60 @@ const MASTER_VIEW_CONTRACT = {
 
 const MOONBEAM_MARKETS = {
   GLMR: {
-    pTokenMarketAddress: '0xdC427cDB81E1532747BEDeB8077a4AEcDbfB585e',
+    pTokenMarketAddress: '0x53d5a47bb874eE688acb479676aD133d47CB9B25',
```

Comment:
what if we track both markets? the key for new markets be {TOKEN}_v1_6 ?

Reply:
Updates to track current and past markets